SONAR-netCDF4/docs/tablePosition_sub_group.adoc

```diff
@@ -0,0 +1,62 @@
+|Description |Obligation |Comment
+|Group attributes | |
+|:string |O |System description
+|Dimensions | |
+|time | |can be fixed or unlimited length, as appropriate
+|Coordinate variables | |
+|uint64 time(time) | |Time from position sensor
+|:axis = "T" | |
+|:calendar = "gregorian" | |
+|:long_name =  "Timestamps for position data" | |
+|:standard_name = "time" | |
+|:units = "nanoseconds since 1601-01-01 00:00:00Z" | |
+| | |
+|Variables | |
+|double latitude(time) |M |latitude of the platform reference point in WGS-84 reference system
+|double :valid_range = –90.0, 90.0 | |
+|:standard_name = "Platform latitude" | |
+|:units = "degrees_north" | |
+|:long_name = "latitude" | |
+|double:_FillValue = Double.NaN | |
+| | |
+|double longitude(time) |M |latitude of the platform reference point in WGS-84 reference system
+|double :valid_range = –180.0, 180.0 | |
+|:standard_name = "Platform longitude" | |
+|:units = "degrees_east" | |
+|:long_name = "longitude" | |
+|double:_FillValue = Double.NaN | |
+| | |
+|float heading(time) |MA |Heading refers to the direction a platform is pointing. This may or may not be the direction that the platform actually travels, which is known as its course or track. Any difference between course and heading is due to the motion of the underlying medium, the air or water, or other effects like skidding or slipping. Heading is typically based on compass directions, so 0° (or 360°) indicates a direction toward true North, 90° indicates a direction toward true East, 180° is true South, and 270° is true West.  
+|:standard_name = "platform_orientation" | |
+|:units = "degree" | |
+|:long_name = "Platform heading(true)" | |
+| | |
+|float course_over_ground(time) |O |a platform course is the cardinal direction along which the platform or aircraft is to be steered
+|:standard_name = "platform_course" | |
+|:units = "m/s" | |
+|:long_name = "degree" | |
+| | |
+|float speed_over_ground(time) |MA |Speed is the magnitude of velocity. The platform speed with respect to ground is relative to the solid Earth beneath it, i.e. the sea floor for a ship.  
+|:standard_name = "platform_speed_wrt_ground" | |
+|:units = "m/s" | |
+|:long_name = "speed_over_ground" | |
+|float :valid_min = 0.0 | |
+| | |
+|float speed_relative(time) |MA |Platform speed relative to water.
+|:long_name = "Platform speed relative to water" | |
+|:standard_name = "platform_speed_wrt_seawater" | |
+|:units = "m/s" | |
+|float :valid_min = 0.0 | |
+| | |
+|float height_above_reference(time) |MA |height of the platform reference point above reference coordinate system (could be geoid, ellipsoid, or chart datum specified in vertical reference attribute)
+|:grid_mapping = "vertical_reference" | |
+|:units = "m" | |
+|:long_name = "height_above_reference" | |
+| | |
+|int vertical_reference(time) |MA |see 5.6. Horizontal Coordinate Reference Systems, Grid Mappings, and Projections in cf-convention 
```

Comment:
The description seems to be copied from somewhere, and the reference to 5.6 is then not useful.

Reply:
This was a reference to the CF convention, explaining the system reference used. More specifically the lat/lon are pretty clear everything is refered to WGS84 but the height_above_reference need to define its reference coordinate system. I planned to add a variable for this, that's why we have height_above_reference:grid_mapping=vertical_reference attribute.

Usually, we could find height relative to ellipsoid or eight relative to geoid (mean sea level). The purpose of this vertical_reference variable was to define which coordinate system is height referred to. Thanks to your remark I just realized that there is no explicit convention for vertical coordinates. 
I can suggest to change height_above_reference to height_above_ellipsoid and add a variable height_above_geoid. I think at least most system will deliver at least elevation relative to ellipsoid (In NMEA protocol it is defined relative to MSL (geoid) with field to refer back to ellipsoid). But we can let the sensor use one or the other (or both).
What do you think of this proposal ?

Reply:
I'm a bit out of my depth with this stuff. As long as it follows some conventions, is easy for non-experts to understand, and there is a diagram/figure that explains it, I'm happy :)

Reply:
I think height_above_ellipsoid and height_above_geoid is explicit, easy to understand and follow convention so I will create those to variables

Reply:
Just pushed my changes. I created to variables : altitude and height_above_reference_ellipsoid to be close to the CF standards. (http://cfconventions.org/Data/cf-standard-names/65/build/cf-standard-name-table.html)